lua/wikis/marvelrivals/Infobox/Character/Custom.lua

```diff
@@ -36,9 +62,18 @@ end
 ---@return Widget[]
 function CustomInjector:parse(id, widgets)
 	local args = self.caller.args
-	if id == 'custom' then
+	if id == 'role' then
+		return WidgetUtil.collect(
+			Cell{
+				name = 'Role',
+				children = self.caller:_getRole(args)
+			}
+		)
+	elseif id == 'custom' then
 		Array.appendWith(
 			widgets,
+			Cell{name = 'Revealed Date', content = {args.revealdate}},
+			Cell{name = 'Game ID', content = {'[[Hero ID|'.. args.gameid .. ']]'}},
```

Comment:
Errors if gameid is not set

Reply:
Considering all hero pages will have a game id (eventually when we make pages for the NPC characters that we have information for, even they have in game IDs), i almost feel like the cell erroring if its not entered is expected.

Reply:
if you want it to error imho make it an intentional error with a proper error message
something like:
```lua
assert(args.gameid, 'missing "|gameid=" input')
```